tests/foreman/ui/test_host.py

```diff
@@ -2387,3 +2387,52 @@ def test_positive_tracer_enable_reload(tracer_install_host, target_sat):
         )
         tracer = session.host_new.get_tracer(tracer_install_host.hostname)
         assert tracer['title'] == "No applications to restart"
+
+
+def test_positive_host_registration_with_non_admin_user(
+    test_name,
```

Comment:
Non-blocking you can remove this i think.
```suggestion

```

Reply:
I would like to keep this.